### DISCUSSION TITLE
Websocket server in `standalone` mode

### DISCUSSION THREAD

**Discussion by sneko** [vercel/next.js#62112] [Help]
### Summary

Hi,

I saw examples where people:
* In development uses Next.js, and aside run another program to handle the websocket server (since Next.js hot reload already does things that could conflict)
* In production, they use a custom `server.js` to reuse the Next.js server when creating the Websocket server (using the same port)

In the past I was using `output: standalone` to have a minimal code, but does any of you have an example with a websocket server? Something straightforward, not hacky things?

Thank you,

**Comment by MasterProvider** [comment]
I hope so they support Websocket on Route API

**Comment by cs-miller** [comment]
it is a bit confusing, i tried to hack something together using `getRequestHandlers` from `next/dist/server/lib/start-server`, but so far its not exactly working
